internal/cmd/loadbalancer/load_balancer.go

```diff
@@ -15,29 +16,41 @@ func NewCommand(s state.State) *cobra.Command {
 		TraverseChildren:      true,
 		DisableFlagsInUseLine: true,
 	}
+	cmd.AddGroup(&cobra.Group{ID: "general", Title: "General"})
+	cmd.AddGroup(&cobra.Group{ID: "protection", Title: "Protection"})
+	cmd.AddGroup(&cobra.Group{ID: "target", Title: "Targets"})
+	cmd.AddGroup(&cobra.Group{ID: "service", Title: "Services"})
+	cmd.AddGroup(&cobra.Group{ID: "network", Title: "Network"})
+	cmd.AddGroup(&cobra.Group{ID: "public-interface", Title: "Public Interface"})
+
 	cmd.AddCommand(
-		CreateCmd.CobraCommand(s),
-		ListCmd.CobraCommand(s),
-		DescribeCmd.CobraCommand(s),
-		DeleteCmd.CobraCommand(s),
-		UpdateCmd.CobraCommand(s),
-		LabelCmds.AddCobraCommand(s),
-		LabelCmds.RemoveCobraCommand(s),
-		AddTargetCmd.CobraCommand(s),
-		RemoveTargetCmd.CobraCommand(s),
-		ChangeAlgorithmCmd.CobraCommand(s),
-		UpdateServiceCmd.CobraCommand(s),
-		DeleteServiceCmd.CobraCommand(s),
-		AddServiceCmd.CobraCommand(s),
-		EnableProtectionCmd.CobraCommand(s),
-		DisableProtectionCmd.CobraCommand(s),
-		AttachToNetworkCmd.CobraCommand(s),
-		DetachFromNetworkCmd.CobraCommand(s),
-		EnablePublicInterfaceCmd.CobraCommand(s),
-		DisablePublicInterfaceCmd.CobraCommand(s),
-		ChangeTypeCmd.CobraCommand(s),
-		MetricsCmd.CobraCommand(s),
-		SetRDNSCmd.CobraCommand(s),
+		util.WithGroup("general", ListCmd.CobraCommand(s)),
+		util.WithGroup("general", DescribeCmd.CobraCommand(s)),
+		util.WithGroup("general", DeleteCmd.CobraCommand(s)),
+		util.WithGroup("general", UpdateCmd.CobraCommand(s)),
+		util.WithGroup("general", LabelCmds.AddCobraCommand(s)),
+		util.WithGroup("general", LabelCmds.RemoveCobraCommand(s)),
+		util.WithGroup("general", ChangeAlgorithmCmd.CobraCommand(s)),
+		util.WithGroup("general", ChangeTypeCmd.CobraCommand(s)),
```

Comment:
These two are actions in the API and from their functionality more powerful than CRUD for other resources. But adding new groups for each one also does not make sense. 

@jooola @phm07 Any suggestions?

Reply:
We already talked about it, it doesn't really fit into Additional Commands but also doesn't justify its own categories, so I put it into General for now (Maybe you could count it as update?). Not sure either though.

Reply:
Keeping them in "General" is fine by me :) Just wanted to bring it up for discussion.